lesson16/main.js

```diff
@@ -188,14 +230,31 @@ function withinThreeDays(day) {
   return diff < msInThreeDays;
 }
 
-function addClickEventListener() {
-  tabArea.addEventListener("click", (e) => {
-    const targetElement = e.target;
-    if (targetElement === e.currentTarget) return;
-    tabArea.querySelector(".active").classList.remove("active");
-    targetElement.classList.add("active");
-    articleArea.querySelector(".active").classList.remove("active");
-    document.getElementById(targetElement.dataset.id).classList.add("active");
+function addClickEventChangeElement(
+  parentElem,
+  isChangeClickedElem,
+  parentOfRelationElem
+) {
+  parentElem.addEventListener("click", (e) => {
+    if (e.target === e.currentTarget) return;
+    if (isChangeClickedElem) {
+      parentElem.querySelector(".active").classList.remove("active");
+      e.target.classList.add("active");
+    }
+    if (parentOfRelationElem.querySelector(".active")) {
+      parentOfRelationElem.querySelector(".active").classList.remove("active");
+    }
+    if (document.getElementById(e.target.dataset.id)) {
+      document.getElementById(e.target.dataset.id).classList.add("active");
+    }
+  });
+}
+
+function addClickEventHideComment() {
+  tabArea.addEventListener("click", () => {
+    if (commentArea.querySelector(".active")) {
+      commentArea.querySelector(".active").classList.remove("active");
+    }
   });
 }

```

Comment:
Q. Does it need to be a function?
```suggestion
tabArea.addEventListener("click", () => {
  if (commentArea.querySelector(".active")) {
    commentArea.querySelector(".active").classList.remove("active");
  }
});

```

Reply:
Thank you sae-san :）
I fixed that.
16705ed1982af4f68f6fdbc6aee9f540b0ce1454